packages/shared/src/components/RecommendedTags.tsx

```diff
@@ -15,7 +15,7 @@ export const RecommendedTags = ({
     return (
       <div>
         <ElementPlaceholder className="mb-3 h-4 w-1/5 rounded-12" />
-        <div className="flex gap-2">
+        <div className="flex flex-wrap gap-2">
```

Comment:
This was intentional? It's a scrollable list actually.

Reply:
I think it became scrollable in one of the recent merged PRs, not really bringing any issues but will remove since no longer neccessary